example/lib/config_object.dart

```diff
@@ -30,6 +30,11 @@ class CountlyConfiguration {
   };
 
   static CountlyConfig getConfig() {
+
+    if(SERVER_URL == 'https://your.server.ly' || APP_KEY == 'YOUR_APP_KEY'){
```

Comment:
If the user does not use this class (e.g if he simply calls `CountlyConfig(SERVER_URL, APP_KEY)..setLoggingEnabled(true)` directly), then no checks will be done.

I feel the checks should be done inside `CountlyConfig`.

Reply:
But this change is intended for example apps only